src/main/scala/roguelike/Math.scala

```diff
@@ -0,0 +1,86 @@
+package roguelike
+
+import indigo._
+
+import scala.math.sqrt
+
+object Math:
```

Comment:
This looks good. In terms of moving it into Indigo I think it's a strong possibility - the extensions too, but thy may warrant some more thought.

We do have all this stuff, but it's part of the `Signal`s based animation process, and this is a totally valid and complementary set of tools.

Reply:
So as it turns out, `Math` is of course a standard Scala class. Renamed to something none conflicting in PR #18, but if we wanted to merge this into Indigo it'll need more thought.